### PR TITLE
Add Enable Web Console ansible play

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: job_templates/service_action_-_enable_web_console.erb
Comment (ekohl) [inline job_templates/service_action_-_enable_web_console.erb]: Various distros have started to use `python2-` or `python3-` prefixes. Does this work on EL8 for example?

File: job_templates/service_action_-_enable_web_console.erb
Comment (ekohl) [inline job_templates/service_action_-_enable_web_console.erb]: Why is a daemon reload needed? Usually the package takes care of that.

File: job_templates/service_action_-_enable_web_console.erb
Comment (ekohl) [inline job_templates/service_action_-_enable_web_console.erb]: Would it make sense to add a parameter to do this? Not everyone is using firewalld.

File: job_templates/service_action_-_enable_web_console.erb
Comment (adamruzicka) [inline job_templates/service_action_-_enable_web_console.erb:7]: Could you add `feature: ansible_enable_web_console` somewhere around here?

**Comment by lzap** [comment]
Let's put this one on hold, I haven been informed in the YT video comment (https://www.youtube.com/watch?v=Ua3b79UG9JM) that Cockpit does NOT need to be enabled on the target host actually. Let's confirm that.

**Comment by adamruzicka** [comment]
It may not need to be enable but as far as I know it needs to be at least installed there. Shouldn't we do at least that?

**Comment by lzap** [comment]
That's a good point, I can confirm that `cockpit` package needs to be enabled for the integration. The service or port is not actually needed.

File: job_templates/service_action_-_enable_web_console.erb
Comment (lzap) [reply job_templates/service_action_-_enable_web_console.erb:7]: Done.

**Comment by lzap** [comment]
With that in mind, the Ansible is now much more simple :-)

**Comment by ekohl** [comment]
Consider my review dismissed/addressed. I'll leave it up to @adamruzicka to ack.

**Comment by lzap** [comment]
I've tested that cockpit-system is the only dependency needed. It also appears to be present on Debian systems. Labeller test do test tho.

**Comment by ekohl** [comment]
@adamruzicka mind having another look?

**Review by adamruzicka — APPROVED** [review]
LGTM

**Comment by ekohl** [comment]
Thanks!

**Comment by ares** [comment]
@lzap mind to open a request to document this? It's a small thing but really useful.

**Pull request merged**
